### DISCUSSION TITLE
How do I change the language to Portuguese

### DISCUSSION THREAD

**Discussion by PabloMelo11** [requarks/wiki#3256] [Error / Bug Report]
What's up guys. I would like to know how I can be changing the language to Portuguese. In the language screen you only have the option 'english'. Where can I include the Portuguese language?

OS: Ubunto 20
Wiki.js version: 2.5.170
Database engine: MySQL

![Screenshot_20201123_115520](https://user-images.githubusercontent.com/38293087/99977230-1dde3680-2d83-11eb-92f8-52c853ff0ab8.png)

**Comment by NGPixel** [comment]
Is your server connected to the internet? The server needs to be able to download the list of available locales....

**Comment by PabloMelo11** [comment]
Yes, the application is connected to the internet.
Is there a way to include the language manually?

**Comment by Smankusors** [comment]
> Yes, the application is connected to the internet.
> Is there a way to include the language manually?

the screenshot you given suggest that the Wiki.js are not able to download the list of available locales. Perhaps it's some firewall rules or your network configuration problem 🤔

**Comment by sya1985** [comment]
Is there no way to deploy language packs offline?

**Reply by NGPixel** [reply]
Have you read the docs?

https://docs.requarks.io/install/sideload
